src/components/MentionListItem/index.tsx

```diff
@@ -90,13 +95,34 @@ export const MentionListItem = ({
             <Text>@{user_data.screen_name}</Text>
           </Box>
         </HStack>
-        <Button
-          variant="outline"
-          onClick={() => onAddItem(user_data)}
-          colorScheme="primary"
-          rounded="full"
-          size="sm"
-        >{t`post-share.add`}</Button>
+        <ButtonGroup>
+          {onRemoveItem && (
+            <Tooltip label={t`post-share.remove`}>
```

Comment:
I don't why, but there is a problem here. Translate remove item doesn't work. Can you check it? And also, Locales add is working.